SDKBuildScripts/mac_unity_RunAutoTests.sh

```diff
@@ -0,0 +1,155 @@
+#!/bin/bash
+
+# USAGE: unity_RunAutoTests.bat [<UNITY_VERSION>] [<SdkName>] [<ProjRootPath>] [<BuildIdentifier>]
+# Make folder links from the UnitySdk to this test project
+# Requires mklink which may require administrator
+# Requires the following environment variables:
+#   TestWin32Build - (Optional - Default false if unset) set to "true" to test building Win32, executing, and geting Jenker! results
+#   TestAndroid - (Optional - Default false if unset) set to "true" to build Android APK
+#   TestiPhone - (Optional - Default false if unset) set to "true" to build iOS XCode project
+#   TestWp8 - (Optional - Default false if unset) set to "true" to build Windows Universal 8 vs-sln
+#   TestPS4 - (Optional - Default false if unset) set to "true" to build Sony PS4
+#   TestSwitch - (Optional - Default false if unset) set to "true" to build Nintendo Switch
+#   TestXbox - (Optional - Default false if unset) set to "true" to build Xbox One
+#   BuildMainUnityPackage - (Optional - Default false if unset) set to "true" to build Unity Package for the SDK
+#   UNITY_PUBLISH_VERSION - (Not required if $1 is defined) Versioned Unity executable name (Assumes multiple potential Unity installations, all in your PATH, each uniquely renamed)
+#   EXECUTOR_NUMBER - (Not required if $4 is defined) Automatic Jenkins variable
+
+
+if [ -z $1 ]; then
+    UNITY_VERSION=$UNITY_PUBLISH_VERSION
+else
+    UNITY_VERSION=$1
+fi
+if [ -z $2 ]; then
+    SdkName=UnitySDK
+else
+    SdkName=$2
+fi
+if [ -z "$3" ]; then
+    ProjRootPath="${WORKSPACE}/${UNITY_VERSION}"
+else
+    ProjRootPath="$3"
+fi
+if [ -z $4 ]; then
+    BuildIdentifier=JBuild_${SdkName}_${EXECUTOR_NUMBER}
+else
+    BuildIdentifier=$4
+fi
+
+CheckVars() {
```

Comment:
Probably we can get rid of all these platform flags, since this script is specific to the mac build.